### PR TITLE
Fix building with GCC 13 and GCC 12.x

### PR DESCRIPTION
**Pull request by marxin**
Fixes missing header files.

Fixes: #156

### CODE REVIEW

**Review by ancorgs — APPROVED** [review]
LGTM, let's just wait for the CI.

**Comment by marxin** [comment]
Thanks. Please push it to `openSUSE:Factory` once you merge it.

**Pull request merged**
